.travis.yml

```diff
@@ -34,8 +34,8 @@ env:
     - SPRING_JPA_SHOW_SQL=false
 
     # if JHIPSTER_BRANCH value is release, use the release from NPM
-    - JHIPSTER_REPO=https://github.com/jhipster/generator-jhipster.git
-    - JHIPSTER_BRANCH=master
+    - JHIPSTER_REPO=https://github.com/erikkemperman/generator-jhipster.git
```

Comment:
Sorry but this is a mistake I think

Reply:
Yes and no -- see the discussion on the accompanying PR on the generator repo:
https://github.com/jhipster/generator-jhipster/pull/7428#discussion_r179921288

I know this is annoying to merge this way, but I don't know how else to make Travis pass on mutually dependent PRs across two repos!